_posts/0000-01-02-kubes2020.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"You miss 100% of the shots you don't take" - Wayne Gretzky, made famous by Michael Scott